server/player_service.py

```diff
@@ -48,35 +51,46 @@ def clear_dirty(self):
 
     async def fetch_player_data(self, player):
         async with db.engine.acquire() as conn:
-            result = await conn.execute(
-                'SELECT mean, deviation, numGames FROM `global_rating` '
-                'WHERE id=%s', player.id)
+            sql = select([
+                avatars_list.c.url,
+                avatars_list.c.tooltip,
+                global_rating.c.mean,
+                global_rating.c.deviation,
+                global_rating.c.numGames,
+                ladder1v1_rating.c.mean,
+                ladder1v1_rating.c.deviation,
+                clan.c.tag
+            ], use_labels=True).select_from(
+                login
+                .join(global_rating)
+                .join(ladder1v1_rating)
+                .outerjoin(clan_membership)
+                .outerjoin(clan)
+                .outerjoin(avatars)
+                .outerjoin(avatars_list)
+            ).where(login.c.id == player.id)
+
+            result = await conn.execute(sql)
             row = await result.fetchone()
             if not row:
-                (mean, dev, num_games) = (1500, 500, 0)
-            (mean, dev, num_games) = row[0], row[1], row[2]
-            player.global_rating = (mean, dev)
-            player.numGames = num_games
-            result = await conn.execute(
-                'SELECT mean, deviation FROM `ladder1v1_rating` '
-                'WHERE id=%s', player.id)
-            row = await result.fetchone()
-            player.ladder_rating = (row[0], row[1])
+                return
 
-            ## Clan informations
-            try:
-                result = await conn.execute(
-                    "SELECT tag "
-                    "FROM login "
-                    "JOIN clan_membership "
-                    "ON login.id = clan_membership.player_id "
-                    "JOIN clan ON clan_membership.clan_id = clan.id "
-                    "where player_id =  %s", player.id)
-                row = await result.fetchone()
-                if row:
-                    player.clan = row[0]
-            except (pymysql.ProgrammingError, pymysql.OperationalError):
-                pass
+            player.global_rating = (
+                row[global_rating.c.mean],
+                row[global_rating.c.deviation]
+            )
+            player.numGames = row[global_rating.c.numGames]
+
+            player.ladder_rating = (
+                row[ladder1v1_rating.c.mean],
+                row[ladder1v1_rating.c.deviation]
+            )
+
+            player.clan = row.get(clan.c.tag)
```

Comment:
Can't you just get values from result `row` just accessing them like: `row.tag`,
same way before